src/visualizr/anitalker/config.py

```
@@ -410,0 +1,409 @@
+            self.window_size,
+            self.frame_jpgs,
+            self.lmd_feats_prefix,
+            self.audio_prefix,
+            self.raw_audio_prefix,
+            self.motion_latents_prefix,
+            self.pose_prefix,
+            self.db_name,
+            audio_hz=self.audio_hz,
+        )
+
+    def make_loader(
```

Ruff (0.12.2)

Too many arguments in function definition (6 > 5)

(PLR0913)